src/entropy_src/data/entropy_src.rdl

```diff
@@ -688,8 +688,8 @@ addrmap entropy_src {
             in the Observe FIFO. The RNG data rate is slow enough that firmware should always
             be able to keep up. This register meanwhile provides an additional check to confirm
             that bytes read from the !!FW_OV_RD_DATA register represent contiguous RNG samples.
-            If an overflow event occurs, this bit must be cleared by software.";
-            sw = rw;
+            If an overflow event occurs, this bit is cleared by hardware as soon as the FIFO is emptied.";
+            sw = r;
             onwrite = woclr;
```

Comment:
We'll need to remove the onwrite property

Reply:
Fixed.